### ISSUE TITLE
4.5.5 image missing for ARM64v8

### ISSUE THREAD

**Issue by yosifkit** [crate/docker-crate#197]
`crate:4.5.5` fails to [build on `arm64v8`](https://doi-janky.infosiftr.net/job/multiarch/job/arm64v8/job/crate/), but it just looks like the binaries were not published on https://cdn.crate.io/downloads/releases/cratedb/aarch64_linux/.

Dockerfile part being run:

https://github.com/crate/docker-crate/blob/53f8d6336816b88f09d831c073cab749762c7be6/Dockerfile#L22-L23

Relevant output:
```console
curl: (22) The requested URL returned error: 404 Not Found
```

**Comment by amotl** [comment]
Dear @yosifkit,

thank you very much for making us aware about this detail. Apparently, the tarball for CrateDB 4.5.5 for ARM64 was missing on `cdn.crate.io`, maybe it was a hiccup on our release pipeline. We just submitted a rebuild to make it available for consumption [1].

Will you trigger a rebuild on your end?

With kind regards,
Andreas.

P.S.: We will soon trim the list of active version tags as advised by @tianon at https://github.com/docker-library/official-images/pull/10618#discussion_r677701905. That will probably remove everything <4.6 altogether, so it will be scheduled after we got 4.5.5 sorted.

[1] https://cdn.crate.io/downloads/releases/cratedb/aarch64_linux/crate-4.5.5.tar.gz
